pkg/controllers/modeljob_controller.go

```diff
@@ -20,16 +20,26 @@ import (
 	"context"
 
 	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	modeljobsv1alpha1 "github.com/caicloud/temp-model-registry/api/v1alpha1"
+	modeljobsv1alpha1 "github.com/caicloud/temp-model-registry/pkg/api/v1alpha1"
+)
+
+const (
+	// ControllerName is the name of the controller.
+	ControllerName = "model-registry-controller"
```

Comment:
```suggestion
	ControllerName = "modeljob-operator"
```

Reply:
SGTM, there is another question:
What's name of the deployment? call modeljob-operator too? or xxxxx-controller-manager(the default name is generate by kubebuiler)

Reply:
I think modeljob-operator is better.

Reply:
ok, I will fix it next PR.